backend/cmd/isready/isready.go

```diff
@@ -0,0 +1,53 @@
+package isready
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/teamhanko/hanko/backend/config"
+	"log"
+	"net/http"
+)
+
+func NewIsReadyCommand(config *config.Config) *cobra.Command {
+	return &cobra.Command{
+		Use:   "isready",
+		Args: cobra.OnlyValidArgs,
+		ValidArgs: []string{"admin", "public"},
+		Short: "Health check a service",
+		Long: `Checks if the specified service is healthy. Possible values are "admin" and "public".
+Uses the "/health/ready" endpoint to check if the service is ready to serve requests.
+		`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 1 {
+				log.Fatalf("Please specify a service to check")
+			}
+			service := args[0]
+			address := ""
+			if service == "admin" {
+				address = config.Server.Admin.Address
+			}
+			if service == "public" {
+				address = config.Server.Public.Address
+			}
+			if address[0] == ':' {
+				address = "0.0.0.0" + address
```

Comment:
I tried this out and it works. However to quote wikipedia:
```
RFC 1122 refers to 0.0.0.0 using the notation {0,0}. It prohibits this as a destination address in [IPv4](https://en.wikipedia.org/wiki/IPv4) and only allows it as a source address under specific circumstances.
```
We would use this as a destination here. Better to just use localhost, then.